tests/src/Integration/Action/EntityPathAliasCreateTest.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\EntityPathAliasCreateTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Core\Entity\ContentEntityType;
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\Action\EntityPathAliasCreate
+ * @group rules_actions
+ */
+class EntityPathAliasCreateTest extends RulesEntityIntegrationTestBase {
+
+  /**
+   * The action to be tested.
+   *
+   * @var \Drupal\rules\Core\RulesActionInterface
+   */
+  protected $action;
+
+  /**
+   * The mocked alias storage service.
+   *
+   * @var \PHPUnit_Framework_MockObject_MockObject|\Drupal\Core\Path\AliasStorageInterface
+   */
+  protected $aliasStorage;
+
+
+  /**
+   * The mocked base field definition.
+   *
+   * @var \PHPUnit_Framework_MockObject_MockObject|\Drupal\Core\Field\BaseFieldDefinition
+   */
+  protected $baseFieldDefinition;
```

Comment:
I guess this should not be necessary anymore once you don't access the base fields in the deriver anymore.